### DISCUSSION TITLE
Localisation for logout button is incorrect

### DISCUSSION THREAD

**Discussion by TPGPL** [ppy/osu#24062] [Q&A] [answered]
![](https://github.com/ppy/osu/assets/19843905/cb493f6a-42f7-42a7-9503-acff7d084792)

currently `UserVerificationStrings.BoxInfoLogoutLink` is used as localisation for the `Sign Out` button, which is problematic, as on the web it's currently used as a part of [another string](https://crowdin.com/translate/osu-web/46/en-pl?filter=basic&value=0#1213) making it incorrect in this context

![](https://github.com/ppy/osu/assets/19843905/e7fce2a9-e78b-4781-aaf3-5b057c2d0475)

vs correct translation from the web

![](https://github.com/ppy/osu/assets/19843905/c89b391a-ff3c-4e82-82df-9ac473c0db48)
![](https://github.com/ppy/osu/assets/19843905/c828c019-5465-45fa-af58-1ea5062efeed)

[This string](https://crowdin.com/translate/osu-web/32/en-pl?filter=basic&value=0#971) should be perfectly fine for this matter

**Comment by Theighlin** [comment]
I reported this some days ago at #23979.  Since this discussion is better explained i'm answering that one with this one.

**Comment by monochrome22** [comment, marked as answer]
Well this is duplicate of https://github.com/ppy/osu/discussions/24026.
